metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -277,11 +279,11 @@ func marshalRequestAndLog(printer func([]byte), req *sd_api.CreateTimeSeriesRequ
 	}
 }
 
-func (sink *StackdriverSink) sendOneRequest(req *sd_api.CreateTimeSeriesRequest) {
+func (sink *StackdriverSink) sendOneRequest(req *monitoringpb.CreateTimeSeriesRequest) {
 	startTime := time.Now()
-	empty, err := sink.stackdriverClient.Projects.TimeSeries.Create(fullProjectName(sink.project), req).Do()
+	err := sink.stackdriverClient.CreateTimeSeries(context.Background(), req, gax.WithGRPCOptions())
```

Comment:
No need to write `gax.WithGRPCOptions()` if you don't pass options.
This function is used for passing more options to gRPC layer.

Reply:
Done